website/src/pages/index.js

```diff
@@ -258,9 +258,46 @@ const NewsletterSection = () => {
   );
 };
 
+const PicPoster = () => {
+  const [display, setDisplay] = useState(true);
+  const [theme, setTheme] = useState('light');
+
+  useEffect(() => {
+    addEventListener('click', () => {
+      if (localStorage.getItem('theme') !== theme) {
+        setTheme(localStorage.getItem('theme'));
+      };
+    });
+  }, [theme]);
+
+  if (!display || localStorage.getItem('SHOW_200_CONTRIBUTOR_EVENT_ENTRY')) {
+    return false;
+  };
+
+  const onClose = () => {
+    setDisplay(false);
+    localStorage.setItem('SHOW_200_CONTRIBUTOR_EVENT_ENTRY', 'read');
+  };
+
+  return (
+    <div className="pic-wrapper">
+      <button className="pic-wrapper-close" onClick={onClose}>
+        <svg aria-hidden="true" focusable="false" data-prefix="fas" data-icon="times" class="svg-inline--fa fa-times fa-w-11" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 352 512">
+          <path fill="currentColor" d="M242.72 256l100.07-100.07c12.28-12.28 12.28-32.19 0-44.48l-22.24-22.24c-12.28-12.28-32.19-12.28-44.48 0L176 189.28 75.93 89.21c-12.28-12.28-32.19-12.28-44.48 0L9.21 111.45c-12.28 12.28-12.28 32.19 0 44.48L109.28 256 9.21 356.07c-12.28 12.28-12.28 32.19 0 44.48l22.24 22.24c12.28 12.28 32.2 12.28 44.48 0L176 322.72l100.07 100.07c12.28 12.28 32.2 12.28 44.48 0l22.24-22.24c12.28-12.28 12.28-32.19 0-44.48L242.72 256z"></path>
+        </svg>
+      </button>
+      <a href="/blog/2021/07/06/celebrate-200-contributors" onClick={onClose}>
+        {theme === "light" && <img src="/static/img/pic_poster/200_mark_light.jpg" alt="" />}
```

Comment:
Put `200_mark_light.jpg` inside the src folder in the asset directory, or serve it by cdn.
As you can see - https://deploy-preview-414--apisix.netlify.app/static/img/pic_poster/200_mark_light.jpg leads to 404 after build.